geometry_msgs/Twist twist
float32 distance